inc/fields/media.php

```diff
@@ -35,6 +35,7 @@ public static function add_actions() {
 		// Print HTML templates for Customizer.
 		// Must called here instead of inside html() because content of the field is injected via JavaScript.
 		add_action( 'customize_controls_print_footer_scripts', [ get_called_class(), 'print_templates' ] );
+		add_action( 'admin_footer', [ get_called_class(), 'print_templates' ] );
```

Comment:
Em thêm comment này vào trước dòng này nhé:

// Must called here instead of inside html() because when creating a custom block, the field is outputed via Ajax.